test/metabase/automagic_dashboards/core_test.clj

```diff
@@ -185,8 +185,8 @@
   (mt/with-test-user :rasta
     (automagic-dashboards.test/with-dashboard-cleanup
       (doseq [[table cardinality] (map vector
-                                       (t2/select Table :db_id (mt/id) {:order-by [[:id :asc]]})
-                                       [15 11 7 5])]
+                                       (t2/select Table :db_id (mt/id) {:order-by [[:name :asc]]})
```

Comment:
There are 2 things happening here:
- this is a consequence of test tables ids being changed ([context](https://github.com/metabase/metabase/pull/36677/files#r1427855029)).
- after https://github.com/metabase/metabase/pull/35973, the test-data database now has 8 tables. so that's why you see my changes also include 4 more items.